pandas-stubs/plotting/_core.pyi

```diff
@@ -77,7 +81,161 @@ def boxplot(
 
 class PlotAccessor(PandasObject):
     def __init__(self, data) -> None: ...
-    def __call__(self, *args, **kwargs): ...
+    @overload
+    def __call__(
+        self,
+        *,
+        data: Series | DataFrame | None = ...,
+        x: Hashable = ...,
+        y: Hashable | Sequence[Hashable] = ...,
+        kind: Literal[
+            "line",
+            "bar",
+            "barh",
+            "hist",
+            "box",
+            "kde",
+            "density",
+            "area",
+            "pie",
+            "scatter",
+            "hexbin",
+        ] = ...,
+        ax: Axes | None = ...,
+        subplots: Literal[False] | None = ...,
+        sharex: bool = ...,
+        sharey: bool = ...,
+        layout: tuple[int, int] = ...,
+        figsize: tuple[int, int] = ...,
+        use_index: bool = ...,
+        title: Sequence[str] | None = ...,
+        grid: bool | None = ...,
+        legend: bool | Literal["reverse"] = ...,
+        style: str | list[str] | dict[HashableT, str] = ...,
+        logx: bool | Literal["sym"] = ...,
+        logy: bool | Literal["sym"] = ...,
+        loglog: bool | Literal["sym"] = ...,
+        xticks: Sequence[float] = ...,
+        yticks: Sequence[float] = ...,
+        xlim: tuple[float, float] | list[float] = ...,
+        ylim: tuple[float, float] | list[float] = ...,
+        xlabel: str = ...,
+        ylabel: str = ...,
+        rot: float = ...,
```

Comment:
docs show `int` for `rot`

Reply:
I'm submitted a PR to pandas to standardized rot and fotnsize to be floats.